eng/targets/Settings.props

```diff
@@ -51,25 +51,65 @@
     <PublishWindowsPdb>false</PublishWindowsPdb>
     <EnforceExtendedAnalyzerRules>false</EnforceExtendedAnalyzerRules>
     <EnforceExtendedAnalyzerRules Condition="'$(IsAnalyzer)' == 'true'">true</EnforceExtendedAnalyzerRules>
+  </PropertyGroup>
 
+  <!-- 
+    There are effectively four modes that are needed for our source build TFMs and this is where
+    we calculate them
+  -->
+  <Choose>
     <!-- 
-      There are effectively three modes that are needed for our source build TFMs
-        - CI normal leg: this needs to build the union of all TFM we support. That ensures we handle all 
-          diagnostics and analyzers for all the TFM that we support.
-        - CI source build leg: this needs to build the current and previous source build TFM. Both are 
-          necessary as the output of this leg is used in other CI source build legs. Those could be 
-          targeting NetCurrent or NetPrevious hence we must produce both.
-        - Source build the product: this is the all up build of the product which needs only NetCurrent
+      CI source build leg: this needs to build the current and previous source build TFM. Both are 
+      necessary as the output of this leg is used in other CI source build legs. Those could be 
+      targeting NetCurrent or NetPrevious hence we must produce both. 
+          
+      However the toolset package we produce must target NetPrevious. This package gets used as the 
+      bootstrap toolset in other repos doing (1). Those can be using a NetPrevious runtime hence 
```

Comment:
You haven't actually numbered any of these. Is this one 1?

Reply:
I will number them.